WebRoot/templates/prefs/Pages.template

```diff
@@ -2302,89 +2302,105 @@ NOTE: The tables inside of the filter rule sections MUST contain a tbody element
-                                        <input type="checkbox" id="${id}_VACATION_DURATION_ENABLED" tabindex=0>
-                                    </td></tr>
-                        <tr><td>
-                            <table role="presentation" class='ZPropertySheet' cellspacing='6'>
-                                <tr>
-                                    <td align="right"><$=ZmMsg.startLabel$></td>
-                                    <td class='ZmOOODuration' >
-                                        <table role="presentation"><tr>
-                                            <td><input style='height:22px;' type='text' autocomplete='off' size=14 maxlength=10 id='${id}_VACATION_FROM1'></td>
-                                            <td class="miniCalendarButton" id='${id}_startMiniCal'></td>
-                                        </tr></table>
-                                    </td>
-                                    <td id='${id}_VACATION_FROM_TIME' class='ZmOOODuration' ></td>
-                                    <td><div id='${id}_VACATION_DURATION_ALL_DAY' class='ZmOOODuration' ></div></td>
-                                </tr>
-                                <tr>
-                                    <td align="right"><$=ZmMsg.endLabel$></td>
-                                    <td class='ZmOOODuration'>
-                                        <table role="presentation"><tr>
-                                            <td><input style='height:22px;' type='text' autocomplete='off' size=14 maxlength=10 id='${id}_VACATION_UNTIL1'></td>
-                                            <td class="miniCalendarButton" id='${id}_endMiniCal'></td>
-                                        </tr></table>
-                                    </td>
-                                    <td colspan="2" class='ZmOOODuration' id='${id}_VACATION_UNTIL_TIME'></td>
-                                </tr>
-                                <tr>
-                                    <td align="right"><$=ZmMsg.vacationCalMsg$></td>
-                                    <td colspan="3">
-                                        <table role="presentation"><tr>
-                                            <td class='ZmOOOCalendar'><div id='${id}_VACATION_CALENDAR_ENABLED' tabindex=0></div></td>
-                                            <td class='ZmOOOCalendar'><div id='${id}_VACATION_CALENDAR_TYPE' tabindex=0 size='3' /></td>
-                                        </tr></table>
-                                    </td>
-                                </tr>
-                                </table>
-                            </td></tr>
-                            </table>
-                            </td>
-                        </tr>
-                    </table>
-                </td>
-            </tr>
-        </table>
+	<table role="presentation" class='ZOptionsSectionTable ZmOutOfOfficeWrapper' width=100%>
```

Comment:
Please see: https://github.com/Zimbra/zm-web-client/pull/96/files#diff-953f1692b15ba4b991582d9b98cc3fb4 for code changes. 

Ignore formatting changes in that commit. As it will be take care after reviewed and will be similar to https://github.com/Zimbra/zm-web-client/pull/96/commits/5408c8738efbdf60e34ab4690dc159bf0fbf74f1#diff-953f1692b15ba4b991582d9b98cc3fb4L2305

Let me know if you have any question. 

Thanks